packages/flutter/lib/src/services/text_boundary.dart

```diff
@@ -150,17 +150,16 @@ class ParagraphBoundary extends TextBoundary {
       return 0;
     }
 
-    final List<int> codeUnits = _text.codeUnits;
     int index = position;
 
-    if (index > 1 && codeUnits[index] == 0xA && codeUnits[index - 1] == 0xD) {
+    if (index > 1 && _text.codeUnitAt(index) == 0xA && _text.codeUnitAt(index - 1) == 0xD) {
```

Comment:
while we're here would you mind making it 0x0A and 0x0D? that's the more common way of expressing these codepoints.

Reply:
(also lower down)

Reply:
Will do